app/controllers/setup_controller.rb

```diff
@@ -0,0 +1,16 @@
+class SetupController < ApplicationController
```

Comment:
I would call this `SetupInstructionController` or similar - worried that a reader might think this is _us_ setting up things at first glance.